### ISSUE TITLE
Binding way makes CDN hard. 

### ISSUE THREAD

**Issue by Sarazh** [jsiebern/bs-material-ui#19]
We tried to use material-ui from CDN, but current binding way makes it impossible to do it. The `default` could not be found. 
We are using 
```
[@bs.module "@material-ui/core"]
external reactClass : ReasonReact.reactClass = "AppBar";
``` 
instead of 
```
[@bs.module "@material-ui/core/AppBar/AppBar"]
external reactClass : ReasonReact.reactClass = "default";
```
Not sure it is an issue or not. Please close it if it is not.

**Comment by jsiebern** [comment]
Yeah, you're right, I'll try to fix that and will put it in an `@next` release to see if it works for everyone! (I think I was having trouble with compilation to commonjs when doing this before, but maybe that has resolved since!)

**Comment by Sarazh** [comment]
Ah, thank you. I think the problem has resolved. I ran with `es6`and `commonjs`, both work.

**Comment by jsiebern** [comment]
Could you kindly check `@next` and let me know if this issue can be closed?

**Comment by jsiebern** [comment]
Fixed in `0.4.2`
